Makefile.system

```diff
@@ -319,7 +319,6 @@ ifeq ($(GCCVERSIONGTEQ7),1)
 else
 	GCCDUMPVERSION_PARAM := -dumpversion
 endif
-GCCMINORVERSIONGTEQ1 := $(shell expr `$(CC) $(GCCDUMPVERSION_PARAM) | cut -f2 -d.` \>= 1)
```

Comment:
This was just re-added in #2955 - should it really be removed again?

Reply:
certainly not, it is just that the remote worktree is/was detached and out of date